src/index.js

```diff
@@ -144,7 +154,13 @@ export class WT {
       delete this.loaderImage.onload;
       reject();
     };
-    this.loaderImage.src = `${this.getUrl()}?${query}`;
+    const params = new this.context.URLSearchParams(query);
+    const sendBeaconResult = this.context.navigator.sendBeacon(`${this.getBeaconUrl()}`, params);
+    if (sendBeaconResult) {
+      resolve();
+    } else {
+      this.loaderImage.src = `${this.getUrl()}?${query}`;
```

Comment:
in line with the above comment on when we'd fall back to this, i'm concerned we don't have visibility into when/how often this fallback is happening. it's liable to happen frequently in areas where we track text input (putting aside how silly some of our tracking is on that front).

Reply:
yeah, great call-out, was thinking something similar. we could add an extra param in the body (that doesn't need to go into the wt event), something like `fallback: true`, and then see frequency by querying request logs.